### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sixinli**
add info for how to configure classpath

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,34 @@ Plugin that sets up a Python environment for building and running tests using
 Usage
 -----
 
-Apply the plugin to your project and configure the associated extension:
+Apply the plugin to your project:
```
Comment (mnazbro) [inline README.md]: This looks like we should just refer you to the plugins website which tells you how: https://plugins.gradle.org/plugin/com.palantir.python.miniconda

**Comment by sixinli** [comment]
can we have both?

File: README.md (hunk 1 above)
Comment (sixinli) [inline README.md]: it doesnt have the how to launch it thingy, can we have both?

Comment (mnazbro) [inline README.md]: Would it be fine to have two steps:
1. Install it according to this page
2. Use this code to add it to your project

**Pull request merged**